win32ss/gdi/ntgdi/freetype.c

```diff
@@ -741,6 +741,24 @@ SubstituteFontByList(PLIST_ENTRY        pHead,
     return FALSE;
 }
 
+static VOID
+IntUnicodeStringToBuffer(LPWSTR pszBuffer, USHORT cbBuffer, const UNICODE_STRING *pString)
+{
+    USHORT cbLength = pString->Length;
+
+    if (cbBuffer < sizeof(UNICODE_NULL))
+        return;
+
+    if (cbLength + sizeof(UNICODE_NULL) > cbBuffer)
+        cbLength = cbBuffer - sizeof(UNICODE_NULL);
+
+    if (cbLength < sizeof(UNICODE_NULL))
```

Comment:
This check doesn't make sense to me. `pString->Length` shouldn't include a UNICODE_NULL, so checking for it doesn't seem useful.

Reply:
Fixed.